src/NuGet/Microsoft.Net.Compilers.Toolset/Microsoft.Net.Compilers.Toolset.Package.csproj

```diff
@@ -9,9 +9,14 @@
     <DevelopmentDependency>true</DevelopmentDependency>
     <PackageDescription>
       .NET Compilers Toolset Package.
-      Referencing this package will cause the project to be built using the specific version of the C# and Visual Basic compilers contained in the package, as opposed to any system installed version.
+      Referencing this package will cause the project to be built using the C# and Visual Basic compilers contained in the package, as opposed to the version installed with MSBuild.
+
+      This package is primarily intended as a method for rapidly shipping hot fixes to customers. Using it as a long term solution for providing newer compilers on older MSBuild installations is explicitly not supported. That can and will break on a regular basis.
```

Comment:
hotfix is one word?

Reply:
https://en.wikipedia.org/wiki/Hotfix

Reply:
my point was that it is likely a typo. (didn't see an alternative spelling in the link)

Reply:
Oh ha. I thought you were commenting that I had used "hotfix" when in reality you were commenting that I didn't. i'm a bad speller. Let me fix.